boards/arm/deployment_module/deployment_module.dts

```diff
@@ -0,0 +1,267 @@
+/dts-v1/;
+#include <st/f4/stm32f446Xe.dtsi>
+#include <st/f4/stm32f446r(c-e)tx-pinctrl.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "Deployment Module";
+	compatible = "st,stm32f446xx";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+
+		zephyr,console = &uart5;
+        zephyr,shell-uart = &uart5;
```

Comment:
idk if this is a weird GitHub rendering thing with take vs spaces but this line is strangely formatted

Reply:
Weird GitHub rendering but Ill format it